docs/docs/4-guides/3-generate-grid-form-views/5-admin-generator-generate-form/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Admin Generator create Form
+---
+
+The Admin Generator can also generate Form Views.
+
+## Generate Form
+
+To generate a Form with the Admin Generator, you have to create a `.cometGen.ts` file. Those files are used by the Admin Generator to generate different kinds of interfaces (DataGrids or Forms). Create a new file `CustomerForm.cometGen.ts` in the `api/src/customer` directory with the following content:
+
+```typescript
+import { future_FormConfig as FormConfig } from "@comet/cms-admin";
+import { GQLCustomer } from "@src/graphql.generated";
+
+export const CustomerForm: FormConfig<GQLCustomer> = {
+    type: "form",
+    gqlType: "Customer",
+    mode: "all",
+    fields: [
+        {
+            type: "fieldSet",
+            name: "info",
+            collapsible: false,
+            fields: [
+                {
+                    type: "text",
+                    name: "id",
+                    readOnly: true,
+                },
+                {
+                    type: "text",
+                    name: "firstName",
+                },
+                {
+                    type: "text",
+                    name: "lastName",
+                },
+            ],
+        },
+    ],
+};
+```
+
+This configuration is one of the most simplified once and can fore sure be extended with more options.
+
```

Comment:
I'd remove this sentence since it doesn't add value. A reference to the document config could be added at a later point

```suggestion
```

Reply:
applied.